packages/config/src/ConfigBuilder.ts

```diff
@@ -64,7 +64,8 @@ export class ConfigBuilder {
         // Prompt for properties missing from base profile
         if (impConfig.baseProfile != null && opts.getValueBack != null) {
             for (const [k, v] of Object.entries(impConfig.baseProfile.schema.properties)) {
-                if (v.includeInTemplate && v.optionDefinition?.defaultValue == null) {
+                const oldPropValue = opts.mergeConfig?.profiles[impConfig.baseProfile.type]?.properties[k];
+                if (v.includeInTemplate && v.optionDefinition?.defaultValue == null && (oldPropValue == null || oldPropValue === "")) {
```

Comment:
should we `.trim()` the `oldPropValue` when comparing `=== ""`  to account for `"    "` ? 😋

Reply:
I believe based on the team vote above, we don't need to check the old property value to see if it's null or empty, but will always overwrite with the prompted for value 🙂